mmv1/products/metastore/terraform.yaml

```diff
@@ -23,6 +23,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "default"
         vars:
           metastore_service_name: "metastore-srv"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dataproc_metastore_service_cmek"
+        min_version: beta
+        primary_resource_id: "default"
```

Comment:
It might make sense to add `skip_docs` to this example and (optionally, if you think the use case is significant enough) add a new example for documentation (marked with `skip_test`) that doesn't have the cruft related to setting up the test environment.

Reply:
That sounds good, I've updated it.